content/docs/started/getting-started-k8s.md

```diff
@@ -4,84 +4,156 @@ description = "Instructions for installing Kubeflow on your existing Kubernetes
 
-## Deploy Kubeflow
+In this reference architecture, we use [Dex](https://github.com/dexidp/dex) and [Istio](https://istio.io/) for vendor-neutral authentication.
 
-Follow these steps to deploy Kubeflow:
+![platform existing architecture](https://i.imgur.com/OlaN73j.png)
+
+### Prerequisites
+- Kubernetes Cluster with LoadBalancer support.
+
+If you don't have a Kubernetes Cluster, you can create a compliant Kubernetes Engine (GKE) on Google Cloud Platform cluster with the following script:
+
+<details>
+
+<summary>GKE Cluster Creation Script</summary>
+
+```bash
+#!/bin/bash
+
+set -e
+
+# This script uses the gcloud command.
+# For more info, visit: https://cloud.google.com/sdk/gcloud/reference/container/
+
+# Edit according to your preference
+GCP_USER="$(gcloud config list account --format "value(core.account)")"
+GCP_PROJECT="$(gcloud config list project --format "value(core.project)")"
+GCP_ZONE="us-west1-b"
+
+CLUSTER_VERSION="$(gcloud container get-server-config --format="value(validMasterVersions[0])")" 
+CLUSTER_NAME="kubeflow"
 
-1. Download a `kfctl` release from the 
-  [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/).
+############################
+# Create and setup cluster #
+############################
 
-1. Unpack the tar ball:
+gcloud container clusters create ${CLUSTER_NAME} \
+--project ${GCP_PROJECT} \
+--zone ${GCP_ZONE} \
+--cluster-version ${CLUSTER_VERSION} \
+--machine-type "n1-standard-8" --num-nodes "1" \
+--image-type "UBUNTU" \
+--disk-type "pd-ssd" --disk-size "50" \
+--no-enable-cloud-logging --no-enable-cloud-monitoring \
+--no-enable-ip-alias \
+--enable-autoupgrade --enable-autorepair
+
+echo "Getting credentials for newly created cluster..."
+gcloud container clusters get-credentials ${CLUSTER_NAME} --zone=${GCP_ZONE}
+
+echo "Setting up GKE RBAC..."
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user=${GCP_USER}
+```
+
+</details>
+
+### Deploy Kubeflow
+
+Follow these steps to deploy Kubeflow:
+
+1. Download a `kfctl` release from the [Kubeflow releases page](https://github.com/kubeflow/kubeflow/releases/) and unpack it:
 
     ```
     tar -xvf kfctl_<release tag>_<platform>.tar.gz
     ```
 
-1. Run the following commands to set up and deploy Kubeflow. The code below
-  includes an optional command to add the binary `kfctl` to your path. If you 
-  don't add the binary to your path, you must use the full path to the `kfctl` 
-  binary each time you run it.
-
-    ```bash
-    # The following command is optional, to make kfctl binary easier to use.
-    export PATH=$PATH:<path to kfctl in your kubeflow installation>
-
-    export KFAPP=<your choice of application directory name>
-    # Default uses IAP.
-    kfctl init ${KFAPP}
-    cd ${KFAPP}
-    kfctl generate all -V
-    kfctl apply all -V
+   Alternatively, you can easily build the `kfctl` binary yourself:
```

Comment:
Remove the word "easily". It's is tech writing best practice to avoid words like "easily" and "simply", as these words can cause frustration and unhappiness for the reader when the solution turns out not to be simple.

Reply:
This makes a lot of sense, I will rephrase it asap.
Do we have a guide for writing best-practices?

This is great advice and we should definitely document it.
Maybe as a Pull Request template ?

Reply:
Thanks! We do have a style guide here:
https://www.kubeflow.org/docs/about/style-guide/

If you'd like to create a PR to add a section on words like "easily", "simply", and "just do something" that'd be great.

The above Kubeflow style guide suggests a [fuller style guide](https://developers.google.com/style/) which does have a section on [words like "easy"](https://developers.google.com/style/tone#some-things-to-avoid-where-possible). It's probably still worth adding the info to the Kubeflow page, to make it easier to find.